.travis.yml

```diff
@@ -9,6 +9,8 @@ before_install:
   - gem update --system
 before_script:
   - createdb pliny-gem-test
+before_install:
+  - gem update --system
```

Comment:
Would you mind removing this? This has already been addressed with https://github.com/interagent/pliny/commit/ae7dc845184267b81af7b5de56bcb8b42fe56596.